sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusConnectionStringProperties.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+import com.azure.core.amqp.implementation.ConnectionStringProperties;
+
+import java.net.URI;
+
+/**
+ * A utility class that parses a connection string into sections.
+ */
+public final class ServiceBusConnectionStringProperties {
```

Comment:
Add unit tests.

Reply:
This class is basically a wrapper for amqp-core's ConnectionStringProperties.java, so I copied the test cases from there and adapted them.